snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -12,33 +12,212 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Google.Apis.Download;
+using Google.Apis.Storage.v1;
+using Google.Apis.Storage.v1.Data;
+using Google.Apis.Upload;
+using System;
+using System.IO;
 using System.Linq;
+using System.Text;
 using Xunit;
 
 namespace Google.Storage.V1.Snippets
 {
     [Collection(nameof(StorageSnippetFixture))]
     public class StorageClientSnippets
     {
-        private readonly StorageSnippetFixture _fixture;
+        private readonly StorageSnippetFixture fixture;
 
         public StorageClientSnippets(StorageSnippetFixture fixture)
         {
-            this._fixture = fixture;
+            this.fixture = fixture;
         }
 
         [Fact]
-        public void ListObjects()
+        void ListBuckets()
         {
-            var bucketId = _fixture.BucketId;
+            var projectId = fixture.ProjectId;
+
+            // <ListBuckets>
+            var client = StorageClient.Create();
+
+            // List all buckets associated with a project
+            var buckets = client.ListBuckets(projectId);
+            // </ListBuckets>
+        }
+
+        [Fact]
+        void CreateBucket()
+        {
+            var projectId = fixture.ProjectId;
+
+            // <Buckets.Insert>
+            var client = StorageClient.Create();
+
+            // GCS bucket names must be globally unique
+            var bucketName = Guid.NewGuid().ToString();
+
+            // Bucket defined in Google.Apis.Storage.v1.Data namespace
+            var bucket = client.Service.Buckets.Insert(new Bucket { Name = bucketName }, projectId).Execute();
```

Comment:
Will look forward to sending you a pull request with this simplified when I've implemented StorageClient.CreateBucket...